pkg/controller/cloud/pvlcontroller_test.go

```diff
@@ -208,7 +208,8 @@ func TestAddLabelsToVolume(t *testing.T) {
 			labeledCh <- true
 			return true, nil, nil
 		})
-		pvlController := &PersistentVolumeLabelController{kubeClient: client, ebsVolumes: &mockVolumes{volumeLabels: map[string]string{"a": "1"}}}
+
```

Comment:
Can probably remove the entire mockVolumes definition with the FakeCloud.GetLabelsForVolume, depending on how you make that call generic.

Reply:
Ack, nuking mockVolumes, as it is not really needed with the VolumeLabelMap i just added in FakeCloud